src/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const ncNewsApi = axios.create({ baseURL: 'https://nc-news-1fcj.onrender.com/api' });
+
+export const getArticles = (topic, sortBy, orderBy) => {
+    return ncNewsApi.get('/articles', { params: { topic: topic, sort_by: sortBy, order: orderBy }})
+    .then(({ data: { articles } }) => {
+        return articles;
```

Comment:
Remember to write just enough code to complete the ticket. At this stage, there is no need for the params object